src/wp-includes/theme.php

```diff
@@ -3589,8 +3589,16 @@ function _wp_customize_changeset_filter_insert_post_data( $post_data, $supplied_
 function _wp_customize_loader_settings() {
 	$admin_origin = parse_url( admin_url() );
 	$home_origin  = parse_url( home_url() );
+	
+	if(!isset($admin_origin['host'])){
+		$admin_origin['host'] = '';
+	}
+	if(!isset($home_origin['host'])){
```

Comment:
```suggestion
	if ( ! isset( $home_origin['host'] ) ) {
```
Coding standards fixes.